curation_concerns.gemspec

```diff
@@ -45,6 +45,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'dry-struct', '~> 0.1'
   spec.add_dependency 'flot-rails', '~> 0.0.7'
   spec.add_dependency 'highcharts-rails'
+  spec.add_dependency 'hydra-pcdm', '~> 0.9.0'
```

Comment:
Instead of pinning hydra-pcdm in CC, should we update the hydra-works gemspec and let CC's dependence on HW pick up the right version of HP?